index.html

```diff
@@ -557,24 +566,47 @@ <h3>Terminology</h3>
 
           <dt><dfn class="export" data-lt="controller(s)|Controllers">controller</dfn></dt>
           <dd>
-An entity that is [=authorized=] to perform an action with a specific resource,
-such as update a [=controller document=] or use a cryptographic key to generate
-a digital signature.
+            <p>
+An entity that is capable of performing an action with a specific resource,
+such as updating a [=controller document=] or generating a [=proof=] using a [=verification method=].
+            </p>
           </dd>
 
           <dt><dfn class="export">controller document</dfn></dt>
           <dd>
-A set of data that specifies one or more relationships between a
-[=controller=] and a set of data, such as a set of public cryptographic keys.
+            <p>
+A document that contains cryptographic material and lists service
+endpoints that can be used for verifying proofs from, and interacting
+with, the [=controller=] of an identifier.
+            </p>
           </dd>
 
           <dt><dfn data-lt="subjects">subject</dfn></dt>
           <dd>
-The entity identified by the `id` property in a [=controller document=].
-Anything can be a subject: person, group, organization, physical thing, digital
-thing, logical thing, etc.
+            <p>
+An entity that is referred to by the `id` property in a [=controller document=]
+when it is used as a subject in other contexts, such as authentication or
+attestations, e.g., Verifiable Credentials. Anything can be a subject: 
+person, group, organization, physical thing, digital thing, logical thing, etc. 
```

Comment:
Hmmm. The "and" seems to suggest that if there isn't another context, then the subject isn't defined (because the definition is dependent on both a referent and use in a different context.

Lol. The current language actually doesn't help either. Let me try a revision:

```suggestion
An entity that is referred to by the `id` property in a [=controller document=].
When the identifier is used as a subject in other contexts, e.g., Verifiable Credentials, 
then the subject referred to in those contexts, is taken as the Subject of the identifier. 
Anything can be a subject: 
person, group, organization, physical thing, digital thing, logical thing, etc. 
```

Reply:
Perhaps...
```suggestion
An entity that is referred to by the value of the `id` property in a
[=controller document=], which value is also used as a subject in other contexts,
such as authentication or attestations, e.g., Verifiable Credentials. Anything
can be a subject: person, group, organization, physical thing, digital thing,
logical thing, etc. 
```

Reply:
I am a bit bothered by the terminology. The term "subject" has a well-defined meaning, see [VCDM](https://www.w3.org/TR/vc-data-model-2.0/#claims). 

Are we sure the terminology is used correctly? What does this statement mean exactly:

> which value is also used as a subject in other context

Does it mean that the value can also appear as a value of an `id` property in another context? Or does it mean that the value can appear as a value for a different property, e.g., a `controller` property in a Verification Method?

If the former, then the whole statement is redundant: this is generally true for any resource identifier both in the VCDM and in RDF. It does not say anything new, neither normatively nor informally. If the latter, then it is incorrect usage, because in that other context the URL is _not_ used as a subject but, rather, as an object...